.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,41 @@
+---
+name: 🐞 Bug Report
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+#### Summary
+
+<!-- Provide a summary describing the problem you are experiencing. -->
+
+#### Current behavior
+
+<!--
+  What is the current (buggy) behavior? Feel free to add as much information as possible.
+  Also, a screenshot often says more than a thousand words. :)
+  -->
+
+#### How to reproduce
+
+<!--
+Provide steps to reproduce the bug.
+Adding a failing Unit or Functional Test would help us a lot - you can submit one in a Pull Request separately, referencing this bug report.
+-->
+
+#### Expected behavior
+
+<!-- What was the expected (correct) behavior? -->
+
+#### If you encounter an error, please create a logfile using the following command and post the output here
+
+```
+$ embark run --nodashboard --logfile log.txt --loglevel trace
+```
+
+#### Please provide additional information about your system
+
+**OS**:
+**Embark Version**:
+**Node Version**:
+**NPM Version**:
```

Comment:
We also often end up asking the user how they installed node/npm, i.e. so we can recommend they install w/ nvm. Related, we often need to ask how they installed embark, i.e. did they use sudo?

What if we included the following:

```
**Command(s) used to install Node/NPM**:
**Command used to install Embark**:
```

Reply:
Updated!

Reply:
Nice. Should we also ask how Node/NPM were installed?

Reply:
You think this is crucial? I haven't been around when those issues you've mentioned came up. If this was an important information, we totally add this.

Reply:
It seems to come up less often now that our docs recommend installing Node with nvm, so maybe not crucial. We can always add it later if it seems we're frequently having to ask how the user installed node.